tests/lib/rules/jsx-sort-props.js

```diff
@@ -269,8 +269,21 @@ ruleTester.run('jsx-sort-props', rule, {
       errors: [expectedError]
     },
     {
+      code: '<App key={2} b a />',
+      options: reservedFirstAsBooleanArgs,
+      output: '<App key={2} a b />',
+      errors: [expectedError]
+    },
+    {
+      code: '<App b a />',
+      options: reservedFirstAsBooleanArgs,
+      output: '<App a b />',
+      errors: [expectedError]
+    },
+    {
       code: '<App dangerouslySetInnerHTML={{__html: "EPR"}} key={2} b />',
       options: reservedFirstAsBooleanArgs,
+      output: '<App key={2} b dangerouslySetInnerHTML={{__html: "EPR"}} />',
```

Comment:
hmm - with reservedFirst tho, the expected output should be `<App dangerouslySetInnerHTML={{__html: "EPR"}} key={2} b />`, no?